packages/components/src/product-icon/README.md

```diff
@@ -0,0 +1,59 @@
+ProductIcon Component
+=============
+
+ProductIcon component is a React component to display plan or product icon in SVG format.
+
+Please note that this component uses asset imports for generating icon images URLs.
+Most build environments handle the asset imports gracefully out of the box (e.g. `create-react-app`, `calypso-build`).
+If you're using a custom Webpack environment, make sure that you have the [asset imports](https://webpack.js.org/guides/asset-management/#loading-images) set up correctly.
+
+## Usage
+
+```jsx
+import { ProductIcon } from '@automattic/components';
+
+export default function MyComponent() {
+    return (
+        <div className="my-component">
+            <ProductIcon slug="free" />
+            <ProductIcon slug="jetpack-professional" />
+            <ProductIcon slug="jetpack-backup-daily" />
+        </div>
+    );
+}
+
+```
+
+## Props
+
+### `slug`
+
+Plan or product slug. Can be one of the following:
+
+- `free`
+- `blogger`
+- `personal`
+- `premium`
+- `business`
+- `ecommerce`
```

Comment:
There's one more nice-to-have thing: can we prefix these WP.com plans with `wpcom-*`? That would put WP.com and Jetpack on the same level of hierarchy:
```
wpcom-free
wpcom-personal
jetpack-free
jetpack-personal
```

If that breaks all existing references and constants in Calypso and would require some mapping code, then it's not worth it, though.

Reply:
This suggestion makes sense. I've added the prefix in [the latest commit](https://github.com/Automattic/wp-calypso/pull/38253/commits/4254a82ff5e5dcda1084d4828344c88e8346d380).

When it comes to the existing references, we will have to update and adjust them in any case since the existing constants values in Calypso are quite messed up. E.g. for the Free plans we have `free_plan` and `jetpack_free` while Personal is `personal-bundle` and `jetpack_personal`.